jekyll/_cci2/credits.md

```diff
@@ -21,15 +21,9 @@ The right plan for you and your team will depend on several factors:
 - If you want access to different machine-types
 - If you need features such as Docker Layer Caching, concurrent builds, or build history
 
-Let's look at how a Performance plan might use credits. With the Performance plan you have:
+Let's look at how a Performance plan might use credits. In this example, your team is divided into several groups working on different projects; some projects are larger while others are smaller and need less resources from a CI configuration. With credits it's possible to specify exactly where and when you need to maximize machine resources.
 
-- 50,000 credits/month ($30 monthly value)
-- Scale up to 80x concurrency
-- Build on Linux, Windows and macOS
-- Access to larger compute
-- Docker layer caching
-
-In this example, your team is divided into several groups working on different projects; some projects are larger while others are smaller and need less resources from a CI configuration. With credits it's possible to specify exactly where and when you need to maximize machine resources. For example, your team might use a `large` `resource_class` (with 4 vCPUs and 8gb of memory at a rate of 20 credits/minute) to speed up a build for a bigger project, while only using the `small` `resource_class` (1 vCPU, 2gb Memory, 5 credits/minute) for a smaller project that may not ship code as frequently, or where build time is inconsequential.
+For example, your team might use a `large` `resource_class` (4 vCPUs and 8gb of memory) and make use of more credits/minute to speed up a build for a bigger project, while only using the `small` `resource_class` (1 vCPU, 2gb Memory) with less credits/minute for a smaller project that may not ship code as frequently, or where build time is inconsequential.
```

Comment:
I feel like maybe taking out the specifics of what the different resource classes include (CPU and memory) would be good, and instead link to them in the config reference so that's the single point of truth? Or the pricing page if they are listed there?